src/components/draggable-table-tags.tsx

```diff
@@ -81,16 +84,78 @@ interface DraggagleTableDataProps {
   attrTitle: string;
   style?: React.CSSProperties;
   isParent?: boolean;
+  resizeCounter?: number;
 }
 
 export const DraggagleTableData: React.FC<DraggagleTableDataProps>
-                = ({collectionId, attrTitle, children, isParent}) => {
+                = ({collectionId, attrTitle, children, isParent, resizeCounter}) => {
   const {dragOverId, dragSide} = useDraggableTableContext();
   const {style} = getIdAndStyle(collectionId, attrTitle, dragOverId, dragSide);
-
+  const {tableScrollTop, scrollY} = useTableTopScrollTopContext();
+
+  const cellRef = useRef<HTMLTableCellElement | null>(null);
+
+  // HACK!!!
```

Comment:
It might be good later to replace this hack with a prop that is passed with the table level.  This is good for now though.